tensorboard/uploader/uploader_subcommand.py

```diff
@@ -429,6 +431,8 @@ def execute(self, server_info, channel):
         )
         experiment_id = uploader.create_experiment()
         url = server_info_lib.experiment_url(server_info, experiment_id)
+        if self.experiment_url_callback:
```

Comment:
Should be `is not None`. (In principle, a user-supplied callable could
be falsy, and what you’re trying to check is whether one was provided,
not how truthy it is.)

Reply:
A falsy callable!?  Subtle, but ok :)

Reply:
Look no further than thine own backyard:

```python
>>> from absl import flags
>>> fv = flags.FlagValues()
>>> callable(fv) and not fv
True
```

Another, found with a bit of REPL-hacking:

```python
>>> from pygments import style
>>> style.Style._styles.clear()
>>> callable(style.Style) and not style.Style
True
```

And `unittest.mock.call` almost fits the bill: it’s a callable subclass
of `tuple`, so could in principle be falsy, but it always happens to be
non-empty.

If Python makes it possible, someone will have done it! The examples
seem far-fetched now, but if an `is not None` could prevent a debugging
nightmare then it seems justified to me—especially since it’s what you
really mean, anyway.